modules/influxdb/influxdb.go

```diff
@@ -147,9 +116,8 @@ func WithConfigFile(configFile string) testcontainers.CustomizeRequestOption {
 	}
 }
 
-// WithInitDb will copy a 'docker-entrypoint-initdb.d' directory to the container.
-// The secPath is the path to the directory on the host machine.
-// The directory will be copied to the root of the container.
+// WithInitDb returns a request customizer that initialises the database using the file `docker-entrypoint-initdb.d`
```

Comment:
Nit: I think the current comment is not explaining that srcPath is a dir, and it's not mentioning the addition of the wait strategy in case this option is used.

Sorry to be picky 🙏  just want to make sure the new comment completes the previous one.

Reply:
> having to have a specifically named file in a directory isn't a user friendly as it could be.

Totally agreed. Worth it to change that in a follow-up

Reply:
Okay, I’ve added additional information indicating that srcPath is a directory for now. Please take a look once again.